content/guides/authentication/jwt/index.md

```diff
@@ -14,32 +14,32 @@ alias_paths:
 
 # JWT Auth
 
-Server-side authentication using JWT one of the most powerful ways to
+Server-side authentication using JSON Web Tokens (JWT) is the most common way to
 authenticate to the Box API. JWT is an [open standard](https://jwt.io/)
 designed to allow powerful server-to-server authentication.
 
 <ImageFrame border>
   ![The JWT flow](./jwt-flow.png)
 </ImageFrame>
 
-Server-side authentication using JSON Web Tokens (JWT) is only available to
-Custom Apps and Enterprise Integrations. It does not involve a user into the
-authorization flow and as such can be used to act on behalf of any user in an
-enterprise. JWT uses a public/private key pair verify the application's
-permissions.
+Server-side authentication using JWT is only available to the Custom Application
+[app type][app-type]. This authentication method does not require end-user
+interaction and, if granted the proper privileges, can be used
+to act on behalf of any user in an enterprise. 
 
-## JWT Restrictions
+There are two ways you can verify an application's permissions:
 
-Server-side authentication using JWT works by creating a claim on the
-application's server and then signing this using the application's secret key.
-In most cases, the claim is that the server is allowed to act on
-behalf of the Box application.
+- using a public and private key pair
+- using a client id and client secret (Client Credentials Grant)
 
-For this reason every application that uses JWT has an associated [Service
-Account](g://authentication/user-types) that is the default user that it
-authenticates as. This user is an admin-like user and for this
-reason JWT applications require an actual Box admin's approval before they can
-be used.
+To learn more about these options visit our guide on using
+[JWT without SDKs][jwtnosdk]. At this time, our SDKs do not support the Client 
```

Comment:
Recommend pulling "At this time, our SDKs do not support the Client Credential Grant." into a warning message block for extra visibility.